schema-guide.md

```diff
@@ -2798,41 +3474,76 @@ Usage example:
 
 `definitions` and its subkeys should not be used directly in `CITATION.cff` files.
 
-- type: ...
-- required: false
+- type: `string`
+- required: `false`
 
-...
+A telephone number.
 
-Usage example:
+Usage examples:
 
 ```yaml
+authors:
+  - family-names: McClane
+    given-names: John
+    tel: +012-345-6789098
+```
+```yaml
+authors:
+  - name: The Research Software Project
+    tel: +012-345-6789098
 ```
 
 ### `definitions.url`
 
 `definitions` and its subkeys should not be used directly in `CITATION.cff` files.
 
-- type: ...
-- required: false
+- type: `string`
+- required: `false`
 
-...
+The URL of a landing page/website for the software or dataset. Supported URLs start with one of:
 
-Usage example:
+- `https://`
+- `http://`
+- `ftp://`
+- `sftp://`
+
+Usage examples:
 
 ```yaml
+url: "https://citation-file-format.github.io/"
+```
+```yaml
+authors:
+  - name: The Research Software Project
+    url: "https://research-software-project.org"
+```
+```yaml
+references:
+  -
+    name: The Research Software Project
```

Comment:
```suggestion
  - name: The Research Software Project
```
While this would've been valid YAML AFAIK, put on one line to save space (and let people figure out their YAML by reading *their* specs).